src/Models/TableItem.php

```diff
@@ -69,4 +71,61 @@ public function getCMSFields()
 
         return $fields;
     }
+
+    /**
+     * Get a row of cells
+     *
+     * Retrieve cells for this row in the table. Only the number of cells
+     * configured in the containing TableBlock will be returned.
+     *
+     * @return ArrayList
+     */
+    public function getCells()
+    {
+        $cells = [];
+        foreach (range(1, $this->TableBlock->NumberOfColumns) as $cell) {
+            $cells[] = DBField::create_field('HTMLText', $this->{"Cell" . $cell});
```

Comment:
I suggest only using double-quotes (`"`) when 

* You are interpolating something, for example `"Hello $world!"`
* You need an escaped character, such as `"\n"`
* The string contains single quotes; `"Hello, 'world'!"` is easier to read than `'Hello, \'world\'!'`

But in general, go with the simplest version.

For this it should be either `'Cell' . $cell` or `"Cell$cell"`. Decide which is easier to read and go with that. :grinning: 

I put you wrong on this, sorry.